client/app/styles/rjsf/ArrayFieldTemplate.tsx

```diff
@@ -67,8 +68,9 @@ const ArrayFieldTemplate = ({
           </div>
         );
       })}
-      {canAdd && (
+      {canAdd && !disabled && (
         <Button
+          disabled={disabled}
```

Comment:
This line is probably redundant but I added it just incase the button showed up somehow. Initially I had just disabled it using this though I though we should just hide it altogether if the form is disabled.

Reply:
makes sense to me